build/images/scripts/PrepareNode.ps1

```diff
@@ -0,0 +1,120 @@
+<#
+.SYNOPSIS
+Assists with preparing a Windows VM prior to calling kubeadm join
+
+.DESCRIPTION
+This script assists with joining a Windows node to a cluster.
+- Downloads Kubernetes binaries (kubelet, kubeadm) at the version specified
+- Registers wins as a service in order to run kube-proxy and cni as DaemonSets.
```

Comment:
by "cni", do you mean "Antrea Agent" here?

Reply:
Yes, change to `Antrea Agent"`